packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -47,74 +47,81 @@ const Native = ({
             className={classNames('dc-input', {
                 'dc-input--disabled': disabled,
                 'dc-input--error': error,
-                'dc-input--hint': hint,
+                'dc-input--hint dc-datepicker--hint': hint,
```

Comment:
just to double check: do we really need to add two classes with one condition?

Reply:
Yes. I need this dc-datepicker--hint to be there only for date-picker-native since everywhere is sharing the same dc-input --hint, i want this component to be different so i added this dc-datepicker--hint to specify the hint in the datepicker